Editor/CourseController/CourseControllerSceneSetup.cs

```diff
@@ -23,6 +24,12 @@ public override void Setup()
             {
                 courseController.GetOrAddComponent<CourseControllerSetup>().ResetToDefault();
             }
+
+            if (courseController == null)
+            {
+                courseController = GameObject.Find("[COURSE_CONTROLLER]");
+            }
+            Selection.activeGameObject = courseController;
```

Comment:
this should probably be put before lines 23ff as the reset to default should always happen...